x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/export_rules/route.ts

```diff
@@ -45,8 +45,14 @@ export const exportRulesRoute = (
       const siemResponse = buildSiemResponse(response);
       const rulesClient = (await context.alerting).getRulesClient();
       const exceptionsClient = (await context.lists)?.getExceptionListClient();
-      const savedObjectsClient = (await context.core).savedObjects.client;
+      const {
+        getExporter,
+        getClient,
+        client: savedObjectsClient,
+      } = (await context.core).savedObjects;
 
+      const client = getClient({ includedHiddenTypes: ['action'] });
+      const actionsExporter = getExporter(client);
```

Comment:
Generic question about this route -- should we deprecate it? 

The only reference I'm finding is within the old [`exportRules()`](https://github.com/elastic/kibana/blob/5e203633051e7e498f2ee344c6547c6b771ce9a5/x-pack/plugins/security_solution/public/detection_engine/rule_management/api/api.ts#L376) action, and that only has references in tests. Since we've moved over to the `bulk_action` API for all our client-side export actions, is there any need to keep this API other than for compatibility? There are slight differences in functionality as you can specify `file_name` and `exclude_export_details` query params, which the bulk action API doesn't currently support, but just curious here if we should mark as deprecated since there are no internal uses and it would give us the opportunity to drop support down the road if we want.